src/SDKs/StreamAnalytics/StreamAnalytics.Tests/ScenarioTests/TransformationTests.ScenarioTests.cs

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Management.Resources;
+using Microsoft.Azure.Management.Resources.Models;
```

Comment:
@atpham256 please use the latest Resource Manager nuget 1.6.0 to record all your tests.
You are using really old Resource Manager 1.1.0. We are asking anyone to use the latest nugets to record all your tests.